deploy/dev/Jenkinsfile

```diff
@@ -25,6 +28,10 @@ pipeline {
                 }
             }
         }
+        /*
+            Angus: Wow, there's a lot going on in this repo! ArgoCD too?! Hardcore :)
```

Comment:
Quite a lot happening here, including ArgoCD 😄. When I started designing a pipeline to update my image on ArgoCD, it was a steep learning curve. However, the 'yq' command-line tool provided me with a straightforward method to update tags, which was a game-changer.

I totally agree with you on the need for more documentation. For a potential user, the current format can be overwhelming and difficult to navigate. I'm going to prioritize adding comprehensive comments and improving documentation to make it more user-friendly.

Thanks for your suggestion; it's valuable feedback like this that helps me enhance the project.